apps/src/code-studio/pd/teachercon1819_registration/TravelPlans.jsx

```diff
@@ -149,7 +150,7 @@ export default class TravelPlans extends Teachercon1819FormComponent {
           {
             this.props.data.needHotel === 'Yes' &&
             this.props.data.needAda === 'Yes' &&
-            this.largeInputFor("explainAda", {required: false})
+            this.largeInputFor("explainAda")
```

Comment:
is this also getting appropriately required on the server?

Reply:
Yes, it already was. The required just meant that the asterisk was/wasn't showing up